src/renderer/lib/dolphin/useDolphinActions.ts

```diff
@@ -23,6 +23,13 @@ export const useDolphinActions = (dolphinService: DolphinService) => {
     [netplayStatus, playbackStatus],
   );
 
+  const getDolphinUserPath = useCallback(
+    async (dolphinType: DolphinLaunchType): Promise<string> => {
+      return await dolphinService.getUserFolderPath(dolphinType);
+    },
+    [dolphinService],
+  );
+
```

Comment:
Hmm I don't think we don't really need this function. The caller could just call `dolphinService.getUserFolderPath()` directly, right?